go/tasks/plugins/hive/executor.go

```diff
@@ -125,9 +127,9 @@ func InitializeHiveExecutor(ctx context.Context, iCtx core.SetupContext, cfg *co
 		return nil, err
 	}
 
-	for clusterLabel, clusterLimit := range resourceConfig {
-		namespaceWithClusterLabel := resourceNamespace.CreateSubNamespace(core.ResourceNamespace(clusterLabel))
-		if err := iCtx.ResourceRegistrar().RegisterResourceQuota(ctx, namespaceWithClusterLabel, clusterLimit); err != nil {
+	for clusterPrimaryLabel, clusterLimit := range resourceConfig {
+		namespaceWithClusterPrimaryLabel := resourceNamespace.CreateSubNamespace(core.ResourceNamespace(clusterPrimaryLabel))
+		if err := iCtx.ResourceRegistrar().RegisterResourceQuota(ctx, namespaceWithClusterPrimaryLabel, clusterLimit); err != nil {
```

Comment:
If the `RegisterResourceQuota` function doesn't have a logger.Infof, can you add one here?